torch/jit/__init__.py

```diff
@@ -477,9 +492,11 @@ def __getattr__(self, attr):
         return self.execution_engines[attr]
 
 
-def script(fn):
-    ast = torch.jit.frontend.get_jit_ast(fn)
-    graph = _jit_script_compile(ast)
+def script(fn, rcb=None):
```

Comment:
There is no way to use script as an annotation if rcb is set. I am not sure how this is normally handled with python annotations.